test/jdk/javax/swing/JTableHeader/TableRendererTest.java

```diff
@@ -0,0 +1,131 @@
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/* @test
+   @bug 6429812
+   @key headful
+   @requires (os.family == "windows")
+   @library /java/awt/regtesthelpers
+   @build PassFailJFrame
+   @summary Test to check if table is printed without NPE
+   @run main TableRendererTest
+*/
+
+import java.awt.Component;
+import java.awt.Robot;
+
+import javax.swing.UIManager;
+import javax.swing.SwingUtilities;
+import javax.swing.JFrame;
+import javax.swing.JTable;
+import javax.swing.JScrollPane;
+
+import javax.swing.table.AbstractTableModel;
+import javax.swing.table.JTableHeader;
+import javax.swing.table.TableCellRenderer;
+
+public class TableRendererTest{
+    static Boolean PassFailStatus = true;
+    static JFrame frame = null;
+    private static JScrollPane jScrollPane = null;
+    private static JTable table = null;
+
+    public static void main(String[] args) throws Exception {
+        UIManager.setLookAndFeel("com.sun.java.swing.plaf.windows.WindowsLookAndFeel");
+
+        Robot robot = new Robot();
+        SwingUtilities.invokeAndWait(new Runnable() {
+            public void run() {
+                try {
+                    Thread.currentThread().setUncaughtExceptionHandler(new ExceptionCheck());
+                    initialize();
+                    //Adding the Test Frame to handle dispose
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+        });
+
+        robot.delay(2000);
+        robot.waitForIdle();
+
+        if (!PassFailStatus) {
+            frame.dispose();
+            throw new RuntimeException("Test Case Failed. NPE raised!");
+        }
+        System.out.println("Test Pass!");
+        frame.dispose();
+    }
+
+    static void initialize() {
+        frame = new JFrame();
+        frame.setTitle("JFrame");
+        table = new JTable(new MyModel());
+        jScrollPane = new JScrollPane();
+        jScrollPane.setBounds(new java.awt.Rectangle(0,0,350,618));
```

Comment:
```suggestion
        jScrollPane.setBounds(0, 0, 350, 618);
```
You can pass the parameters directly without creating `Rectangle`. Spaces after commas, please.